core/quill.js

```diff
@@ -72,8 +72,9 @@ class Quill {
     this.clipboard = this.theme.addModule('clipboard');
     this.history = this.theme.addModule('history');
     this.theme.init();
-    let contents = this.clipboard.convert(`<div class='ql-editor' style="white-space: normal;">${html}<p><br></p></div>`);
-    this.setContents(contents);
```

Comment:
What do you think about reassigning the `html` variable with the complete contents to shorten line length?

```javascript
let html = this.container.innerHTML.trim();
...
html = `<div class='ql-editor' style="white-space: normal;">${html}<p><br></p></div>`;
let contents = this.clipboard.convert(html);
this.setContents(contents);
```